README.md

```diff
@@ -31,13 +31,26 @@ file [here](examples/unit_test_example.sh).
 The collection of helper libraries are expected to be loaded using the
 provided [src/bootstrap.sh](bootstrap) script.
 
-Currently, there are 2 supported modules. To read more about what each of the modules provide,
+The bootstrap's script responsibilities are to get a local copy of this repository, initialize
```

Comment:
Good content. I have some suggestions for clarity and grammar:

[Usage](https://github.com/RedHatInsights/cicd-tools/blob/7b6915483fa3b2c1dbb800dd483c71c55ee6288e/README.md#bash-script-helper-scripts-usage)

The  [src/bootstrap.sh](https://github.com/RedHatInsights/cicd-tools/blob/7b6915483fa3b2c1dbb800dd483c71c55ee6288e/bootstrap) script pulls a local copy of this repo and initializes the loader module, which serves as an entrypoint into the library. The loader module provides the ` cicd::loader::load_module` functions that enable loading different modules.

See the table below for information on the modules:

Reply:
Thanks, I applied your suggestion ! I have set section name to "Bash script library usage" instead of simply "usage" because at the moment the library is an addition to this repo, the former "bootstrap" script and all other scripts tied to it are still hosted here - the library will overcome this scripts eventually, but not just yet, so I prefer to be explicit about this being a particular section of the repo. I'll leave a simple "Usage" section once the old scripts are refactored and / or deprecated.